check-ostree.yaml

```diff
@@ -899,6 +900,33 @@
       command: journalctl -u fdo-client-linuxapp
       register: result_fdo_client_linuxapp_journalctl
 
+    # case: checking firewall customizations
+    - name: Check applied firewall customizations
+      block:
+        - name: Ensure firewall customizations applied from blueprint in trusted zone
+          command: firewall-cmd --info-zone=trusted
+          register: result_trusted_zone
+          become: yes
+        - name: Ensure firewall customizations applied from blueprint in work zone
+          command: firewall-cmd --info-zone=work
+          register: result_work_zone
+          become: yes
+
+        - assert:
+            that:
+              - "'192.168.100.51' in result_trusted_zone.stdout"
+              - "'192.168.100.52' in result_work_zone.stdout"
+            fail_msg: "No firewall customizations found"
+            success_msg: "Firewall customizations added from blueprint"
+      always:
+        - set_fact:
+            total_counter: "{{ total_counter | int + 1 }}"
+      rescue:
+        - name: failed count + 1
+          set_fact:
+            failed_counter: "{{ failed_counter | int + 1 }}"
+      when: firewall_feature == "true"
```

Comment:
1. The checking should be running on RHEL 8.8/9.2 and above, CS8/9, and Fedora, like https://github.com/virt-s1/rhel-edge/pull/1665/files#diff-e719b01c473575efb6bc5c7f6423d53ddbca736721550cd078e3c9b1a5b154e4R281

Reply:
I saw you did this in ostree.sh script. But this ansible playbook will run on all distros.

Reply:
Variable firewall_feature: "false" should avoid this.